packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.base.tsx

```diff
@@ -423,15 +528,127 @@ export class ScrollablePaneBase extends BaseComponent<IScrollablePaneProps, IScr
     return contentContainer ? contentContainer.offsetHeight - contentContainer.clientHeight : 0;
   }
 
+  private _getScrollbarHeightFromLocalStorage(onlyRead?: boolean): number {
+    let scrollbarHeight: number = 0;
+    if (typeof window !== 'undefined' && 'localStorage' in window) {
```

Comment:
Does the usage of `window.localStorage` pass server-side render tests?

We should probably allow for consumers to pass-in a height rather than have a scrollable component accessing `localStorage` directly.

Reply:
For ssr, window would not be available, it would just return 0. Yes, it is getting passed for ssr tests. 
We can't 've scrollbar height/width to be passed as a prop as it may vary for different browsers.
If scrollbar height/width is to be passed as a prop then scrollbars would 've to styled accordingly.

Reply:
FYI `window.localStorage` helps to save a significant amount of time before and after PLT.
![image](https://user-images.githubusercontent.com/41147089/60882281-22c14980-a265-11e9-9121-ea20c6fb0a61.png)